langchain/src/chat_models/openai.ts

```diff
@@ -94,7 +94,7 @@ function messageToOpenAIRole(message: BaseMessage): OpenAIRoleEnum {
 
 function messageToOpenAIMessage(message: BaseMessage): OpenAICompletionParam {
   const msg = {
-    content: message.content || null,
+    content: (message.content as any) || null,
```

Comment:
Is this any typed because we do not know exactly what the types are yet?

Reply:
Yeah just optimizing for speed

Reply:
Will go back and fix before merging into `main` - linter warnings are good in this case